backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -147,11 +133,17 @@ export class AdminResolver {
     return !!res
   }
 
+  @Authorized([RIGHTS.SEARCH_USERS])
```

Comment:
This seems to be the wrong right - every function gets its own right.

Reply:
OK, I was not sure while implementing it, I understood that SEARCH_USERS is the actual Admin role / right